web/packages/core/src/shadow-template.ts

```diff
@@ -173,15 +161,11 @@ ruffleShadowTemplate.innerHTML = `
             color: #FFAD33;
         }
 
-        #right_click_menu > :first-child {
+        #right_click_menu > :first-child,
+        #right_click_menu > :last-child {
             border-top-right-radius: 5px;
             border-top-left-radius: 5px;
         }
```

Comment:
Looks like this is making the bottom item of the menu have square corners on the bottom when it's highlighted?